randEndpts.js

```diff
@@ -0,0 +1,14 @@
+function randomNum() {
```

Comment:
Remember, we are supposed to be using some kind of skewed distribution.  I think the exponential distribution from  https://www.npmjs.com/package/rands will work [if I'm understanding this correctly](https://www.npmjs.com/package/rands)

Reply:
Cool. I'll look into that.